api/Window.json

```diff
@@ -6004,7 +6004,7 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": false
+              "version_added": "15.0"
```

Comment:
Excellent, this is the last remaining bit of https://github.com/mdn/browser-compat-data/pull/9666 (after https://github.com/mdn/browser-compat-data/pull/11489)